packages/styles/scss/patterns/sub-patterns/content-section/_content-section.scss

```diff
@@ -0,0 +1,83 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '../../../globals/imports';
+
+@mixin themed-items {
+  color: $text-01;
+  background: $ui-background;
+}
+
+@mixin content-section {
+  .#{$prefix}--content-section {
+    &__grid {
+      @include carbon--make-container;
+    }
+    &__row {
+      @include carbon--make-row;
+    }
+    &__left,
+    &__children {
+      @include carbon--make-col-ready;
+    }
+    &__left {
+      @include carbon--breakpoint(lg) {
+        @include carbon--make-col(4, 16);
+      }
+    }
+    &__heading {
```

Comment:
I believe this has a width of `90%`. See about syncing up with @RaphaelAmadeu and see if we can get a mixin or something for this. He's working on it for the Card #1342 .

Reply:
Okay.

Reply:
@photodow updated the PR for above comments except this, will sync up with @RaphaelAmadeu on this.

Reply:
ok, but let's not neglect this.